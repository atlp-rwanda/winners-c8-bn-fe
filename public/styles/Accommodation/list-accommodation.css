.accommodation-wrapper{
align-content: center;
top: 50%;
left: 50%;
margin-top:40px;
/* background-image: url('/public/images/mountain-ice.jpg'); */
background-size: cover;
/* background-color: red; */
}
.accommodations{
    /* background-image: url('/public/images/mountain-ice.jpg'); */
}
.accommodation{
    display: flex;
    margin-bottom: 10px;
    /* border: 1px solid cadetblue; */
    border-top: 1px solid rgb(145, 140, 140);
    /* border-bottom: 1px solid #000; */
    border-radius: 4px;
    padding: 5px;
    align-items: center;
    /* height: 150px; */
}
.accomodation__thumbnail{
    width: 200px;
}
.accomodation__thumbnail img{
    height: 140px;
    width: 200px;
    padding-right: 30px;
}
.accommodations .accomodation__desc{
    width: 250px;
}
.accommodations .accomodation__btn{
    background-color: deepskyblue;
    color: #fff;
    margin-left: 450px;
    font-size: 15px;
    padding: 7px;
    border: none;
    border-radius: 10%;
}
.accomodation__new{
    background-color: deepskyblue;
    color: #fff;
    /* margin-left: 500px; */
    margin: 10px;
    /* font-size: 15px; */
    padding: 7px;
    border: none;
    border-radius: 4px;
}
.pagination{
    float: right;
    /* margin-bottom: 50px; */
}
.pagination div > input{
    font-size: 15px;
    margin-left: 10px;
}

/* single accommodation styling */
.single-accommodation{
    display: flex;
    margin-top:40px;
}
.single-accommodation .rate__btn{
    font-size: 15px;
    padding: 7px;
    border: 1px solid #c4c4c4;
    border-radius: 10%;
}
.single-accommodation .add__btn{
    background-color: deepskyblue;
    color: #fff;
    margin-right: 5px;
    font-size: 15px;
    padding: 7px;
    border: none;
    border-radius: 10%;
}
.single-accommodation .add__btn:hover{
color:deepskyblue;
background-color: #fff;
}