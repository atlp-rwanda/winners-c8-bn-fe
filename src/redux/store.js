// this file will configure the redux's store
import { createStore } from "redux";
import rootReducer from "./reducers";

let authToken = ""; 
if(window.localStorage.getItem("auth-token")) {
  authToken = window.localStorage.getItem("auth-token");
}

const initialState = {
    auth: {token: authToken},
    navbar: {currentPage: "HOME"},
};
const store = createStore(
    rootReducer,
    initialState // sets initial state
  );

export default store;
