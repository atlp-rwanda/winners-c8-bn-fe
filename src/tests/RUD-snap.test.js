import React from 'react';
import { Provider } from 'react-redux';
import Accommodation from '../components/ListAccomodations/Accomodations';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom'; 
import store from "../redux/store";

test('<Accommodation /> matches snapshot', () => {
    const component = renderer
        .create(
            <Provider store={store}>
                <MemoryRouter>
                    <Accommodation />
                </MemoryRouter>
            </Provider>,
        )
        .toJSON();
    expect(component).toMatchSnapshot();
});


