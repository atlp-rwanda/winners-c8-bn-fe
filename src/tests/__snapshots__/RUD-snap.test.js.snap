// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accommodation /> matches snapshot 1`] = `
<div
  className="mainContent"
  data-testid="update-1"
>
  <div
    className="accommodation-wrapper"
  >
    <div
      className="accommodations"
    >
      <ul
        className="paginationBttns"
      >
        <li
          className="previous paginationDisabled"
        >
          <a
            aria-disabled="true"
            aria-label="Previous page"
            className="previousBttn "
            onClick={[Function]}
            onKeyPress={[Function]}
            rel="prev"
            role="button"
            tabIndex="-1"
          >
            Previous
          </a>
        </li>
        <li
          className="next"
        >
          <a
            aria-disabled="false"
            aria-label="Next page"
            className="nextBttn"
            onClick={[Function]}
            onKeyPress={[Function]}
            rel="next"
            role="button"
            tabIndex="0"
          >
            Next
          </a>
        </li>
      </ul>
    </div>
    <div />
    <div />
    <div />
  </div>
</div>
`;
